encoding/av1-break-even-calculator/info.json

```
@@ -0,0 +1,21 @@
+{
+  "title": "AV1 break-even point calculator",
```

```suggestion
  "title": "AV1 break-even calculator",
```
short is beautiful :)